cmd/cli/install_test.go

```diff
@@ -773,29 +768,60 @@ func TestEnforceSingleMeshWithExistingMesh(t *testing.T) {
 	assert.Nil(err)
 
 	install := &installCmd{
-		out:                         out,
-		chartPath:                   testChartPath,
-		containerRegistry:           testRegistry,
-		osmImageTag:                 testOsmImageTag,
-		osmImagePullPolicy:          defaultOsmImagePullPolicy,
-		certificateManager:          "tresor",
-		serviceCertValidityDuration: "24h",
-		prometheusRetentionTime:     testRetentionTime,
-		meshName:                    defaultMeshName + "-2",
-		enableEgress:                true,
-		deployPrometheus:            true,
-		deployGrafana:               false,
-		clientSet:                   fakeClientSet,
-		envoyLogLevel:               testEnvoyLogLevel,
-		controllerLogLevel:          testControllerLogLevel,
-		enforceSingleMesh:           true,
+		out:                     out,
+		chartPath:               testChartPath,
+		containerRegistry:       testRegistry,
+		osmImageTag:             testOsmImageTag,
+		osmImagePullPolicy:      defaultOsmImagePullPolicy,
+		certificateManager:      "tresor",
+		prometheusRetentionTime: testRetentionTime,
+		meshName:                defaultMeshName + "-2",
+		deployPrometheus:        true,
+		deployGrafana:           false,
+		clientSet:               fakeClientSet,
+		controllerLogLevel:      testControllerLogLevel,
+		enforceSingleMesh:       true,
 	}
 
 	err = install.run(config)
 	assert.NotNil(err)
 	assert.Equal(err, errAlreadyExists)
 }
 
+func TestReadMeshConfigFile(t *testing.T) {
```

Comment:
Why is the CLI dealing with meshconfig parsing? Seems like stale code?

Reply:
This was an initial change, where osm CLI can take a MeshConfig manifest file.

Will remove.